cpp/include/cudf_test/type_lists.hpp

```diff
@@ -287,11 +287,23 @@ using FixedWidthTypes = Concat<NumericTypes, ChronoTypes, FixedPointTypes>;
  * Example:
  * ```
  * // Invokes all typed fixture tests for all fixed-width types in libcudf
- * TYPED_TEST_CASE(MyTypedFixture, cudf::test::FixedWidthTypes);
+ * TYPED_TEST_CASE(MyTypedFixture, cudf::test::FixedWidthTypesWithoutFixedPoint);
```

Comment:
Question: Why was this example changed in this PR?

Reply:
Oh I just happened to come across this while I was extending the type lists. I think the example for `FixedWidthTypesWithoutFixedPoint` is not really using the actual type list?